src/shared/components/RadioButton/RadioButton.style.ts

```diff
@@ -18,44 +23,47 @@ export const Input = styled.input`
 const colorFromProps = ({ error, checked, disabled }: RadioButtonStyleProps) => {
```

Comment:
Seems we're missing the hover state? Both for unchecked and checked states (or if we have it, it's not what's in Figma)

Reply:
Removed pseudo-element responsible for that. It's back now. Fixed